website/docs/docs/contributing/adapter-development/6-verifying-a-new-adapter.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Verifying a new adapter"
+id: "6-verifying-a-new-adapter"
+---
+
+## Why verify an adapter?
+
+Circa 2017,  dbt's adapter plugin framework was introduced to meet client demand back when dbt Labs (née Fishtown Analytics) still a data consultancy. As a result dbt Labs specialized in four data platforms and became experts in them. However, the covering the surface area of all possible databases, their respective nuances, and keeping them up-to-date and bug-free is a Herculean and/or Sisyphean task that couldn't be done by a single person or even a single team! Enter the dbt community which enables dbt Core to work on more than 30 different databases (32 as of Sep '22)!
```

Comment:
Yeah, this is true! https://github.com/dbt-labs/dbt-core/pull/259

Also TIL that Redshift was technically first: https://github.com/dbt-labs/dbt-core/pull/174

Feel free to take or leave any of this, I was just having fun:
```suggestion
The very first data platforms dbt supported were Redshift and Postgres. In 2017, back when dbt Labs (née Fishtown Analytics) was still a data consultancy, we added support for Snowflake and BigQuery. We also turned dbt's database support into an adapter framework, and a plugin system a few years later. For years, dbt Labs specialized in those four data platforms and became experts in them. However, the surface area of all possible databases, their respective nuances, and keeping them up-to-date and bug-free is a Herculean and/or Sisyphean task that couldn't be done by a single person or even a single team! Enter the dbt community which enables dbt Core to work on more than 30 different databases (32 as of Sep '22)!
```

Reply:
```suggestion
The very first data platform dbt supported was Redshift followed quickly by Postgres (([dbt-core#174](https://github.com/dbt-labs/dbt-core/pull/174)). In 2017, back when dbt Labs (née Fishtown Analytics) was still a data consultancy, we added support for Snowflake and BigQuery. We also turned dbt's database support into an adapter framework ([dbt-core#259](https://github.com/dbt-labs/dbt-core/pull/259/)), and a plugin system a few years later. For years, dbt Labs specialized in those four data platforms and became experts in them. However, the surface area of all possible databases, their respective nuances, and keeping them up-to-date and bug-free is a Herculean and/or Sisyphean task that couldn't be done by a single person or even a single team! Enter the dbt community which enables dbt Core to work on more than 30 different databases (32 as of Sep '22)!
```